cmd/admission-webhook/main.go

```diff
@@ -32,13 +32,15 @@ var (
 	printVersion             bool
 	extraServiceAccounts     string
 	evictRegionLeaderTimeout time.Duration
+	resyncDuration           time.Duration
 )
 
 func init() {
 	flag.BoolVar(&printVersion, "V", false, "Show version and quit")
 	flag.BoolVar(&printVersion, "version", false, "Show version and quit")
 	flag.StringVar(&extraServiceAccounts, "extraServiceAccounts", "", "comma-separated, extra Service Accounts the Webhook should control. The full pattern for each common service account is system:serviceaccount:<namespace>:<serviceaccount-name>")
 	flag.DurationVar(&evictRegionLeaderTimeout, "evictRegionLeaderTimeout", 3*time.Minute, "TiKV evict region leader timeout period, default 3 min")
+	flag.DurationVar(&resyncDuration, "resync-duration", 12*time.Hour, "Resync time of informer")
```

Comment:
```suggestion
	flag.DurationVar(&minResyncDuration, "min-resync-duration", 12*time.Hour, "The resync period in reflectors will be random between MinResyncPeriod and 2*MinResyncPeriod.")
```

and add this to choose  a random resync period:

```
	// We choose a random resync period between MinResyncPeriod and 2 *
	// MinResyncPeriod, so that our pods starting at the same time don't list the apiserver simultaneously.
	resyncPeriod := time.Duration(minResyncPeriod.Seconds()*(1+rand.Float64())) * time.Second
```

Reply:
This is a common practice to avoid generate the load simultaneously.